
ocs2_msgs/mpc_observation observation
ocs2_msgs/mpc_target_trajectories target_trajectories
ocs2_msgs/mpc_flattened_controller flattened_controller
float32[] obs # obs that goes into the NN